server.rb

```diff
@@ -129,6 +129,12 @@ def admin_get_message
     erb :show_playlist
   end
 
+  get "/about_song/:id" do
+    require_user
```

Comment:
This is fine, but this seems like it'd be alright to see even if you're not logged in.